tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -19,20 +19,28 @@ Feature: spaces management
     Then "Alice" should see the following spaces
       | id     |
       | team.a |
-    And "Alice" updates the quota for space "team.a" to "50"
+    When "Alice" updates the space "team.a" name to "developer team" using the context-menu
+    And "Alice" updates the space "team.a" subtitle to "developer team-subtitle" using the context-menu
+    And "Alice" updates the space "team.a" quota to "50" using the context-menu
+    And "Alice" disables the space "team.a" using the context-menu
+    And "Alice" enables the space "team.a" using the context-menu
```

Comment:
I would suggest to reduce repeated steps:
for example: 
for space team.a we can
- update name
- update subtitle
- update quota
- disable
- enable
- should see team.a

for space team.b:
- disable and delete
- should not see team.b

What do you thing? 
or for space team.b -> updates atribytes by disabled space

Reply:
yes that sounds good. I'll update them accordingly